packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -5,7 +5,6 @@ const defaultConfigs = {
   "ETH/BTC": {
     type: "medianizer",
     pair: "ethbtc",
-    lookback: 7200,
```

Comment:
did you make this a default? why is this not here anymore?

Reply:
`CreatePriceFeed` defaults the lookback to an EMP's `liquidationLiveness` so this wasn't actually doing anything